host/service/bsd/service.go

```diff
@@ -17,7 +17,8 @@ import (
 type Service struct {
 	service.Config
 	service.ConfigFileStorer
-	Path string
+	RcDaemonPath string
+	RcConfigPath string
```

Comment:
Let's keep `Path` named that way so it stays homogeneous with other types of init.
RcConfigPath does not need to be in this struct as the template does not use it. Just use `rcConfPath(s.Name)` where needed.

Reply:
Thanks, fixed.